### DISCUSSION TITLE
关于Rclone移动和复制功能的疑问

### DISCUSSION THREAD

**Discussion by LGuangming** [jxxghp/MoviePilot#1001] [问题咨询]
想问问大佬们这个Rclone的移动复制到底应该怎么使用啊，原来的nastools就没搞明白。首先我在容器内创建了MP的配置文件，也能挂载容器内正常连接。但我的问题是：

1). 我使用Rclone方式转移文件是否还需要挂载MP到容器内文件夹（我理解是需要挂载，不然填写了同步目录会提示找不到路径）。但如果要挂载，那跟我在容器外挂载然后映射到容器内又有什么区别呢？

2).如果需要挂载，那每次容器重启会自动挂载吗?还是需要手动挂载或者脚本自动挂载?

3).假如我成功挂载到容器内，我填写同步目录是填写我挂载到容器的实际路径还是填什么路径呢？

非常感谢有经验的人士帮忙解答一下，谢谢！

**Comment by YanTaocheng** [comment]
1. ssh进入到MoviePilot容器，用rclone config配置你的网盘，名称叫MP

2. 我试了下，重启不需要重新配置rclone

3. 同步目录填的是你网盘内的目录

**Reply by LGuangming** [reply]
我目前已经创建好rclone的配置文件了，名称是MP。然后同步目录配置写的是MP:/webdav/download:MP:/TV#rclone_move
 这两个都是网盘内的路径，提示找不到路径。 

然后我在容器内用rclone mount挂载网盘到mnt目录下，使用mnt/webdav/download:mnt/TV#move就没问题。但这样就等于我挂载到本地使用move模式，而不是rclone_move。这个问题是重启后我需要使用rclone mount重新挂载，不知道是不是我哪里配置有问题。

**Reply by YanTaocheng** [reply]
目录监控 的配置：
/downloads_hdd/电影:/aliyun/Media#rclone_move
前面是docker映射到MP的本地地址，后面是网盘内的路径

**Reply by LGuangming** [reply]
> 目录监控 的配置：
> 
> /downloads_hdd/电影:/aliyun/Media#rclone_move
> 
> 前面是docker映射到MP的本地地址，后面是网盘内的路径

还是没成功😩😩
